setup/README.md

```diff
@@ -160,7 +160,30 @@ are printed bold**, all other fields might be ommitted. A minimal example is
   * *virusscanner* to be used. Either `none`, `sophos`, `antivir` or `clamav`
   * *path_to_scan* command of the scanner
   * *path_to_clean* command of the scanner
-
-
-
- 
+* *privacysecurity*
+  * *https_enabled* always redirects to a https url if value is `1`, default is `0`
+* *webservices*
+  * *soap_user_administration* enable administration per soap
+  * *soap_wsdl_path* path to the ilias wsdl file, default is 'http://<your server>/webservice/soap/server.php?wsdl
+  * *soap_connect_timeout* maximum time in seconds until a connection attempt to the SOAP-Webservice is interrupted
+  * *rpc_server_host* Java-Server host (if set `prc_server_port` must be set too)
+  * *rpc_server_port* Java-Server port (if set `prc_server_host` must be set too)
+* *chatroom*
```

Comment:
Please be aware of the already existing MD file here: https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/Modules/Chatroom/README.md Maybe this MUST be changed as well.

Reply:
My suggestion would be: keep the documentation in the setup as is, but link to the extended README from the chatroom. Then adjust the chatroom README by keeping the current detail. Would this be ok?

Reply:
Yes, this is okay for me.

Reply:
I'll link the extended README into the setup documentation. @mjansenDatabay is ok for you, if we adjust the extended Chatroom README after this PR in a new one.

Reply:
@daniwe4 Yes, this is okay.